contracts/PetPark.sol

```diff
@@ -1,7 +1,114 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address owner;
+
+    // define struct of animal type, age, and gender
+    struct animalAndDemo {
```

Comment:
The general convention is to have the first letter as uppercase in names of structs, classes and events